commands/certmanager/constants.go

```diff
@@ -0,0 +1,37 @@
+package certmanager
+
+// flags
+const (
+	CertId              = "certificate-id"
+	CertName            = "certificate-name"
```

Comment:
We can shorten this to just `name`, which is a common flag (compute, k8s, to be used in mongo).
Move to pkg/constants (if this doesn't exist, rebase from master) 

Will be used as `constants.FlagName`